src/aelfrice/store.py

```
@@ -1961,6 +1976,52 @@ def soft_delete_belief(self, belief_id: str, ts: str | None = None) -> None:
+            SELECT b.*,
+                   (SELECT COUNT(*) FROM belief_corroborations bc
+                    WHERE bc.belief_id = b.id) AS corroboration_count
+            FROM beliefs b
+            WHERE b.valid_to IS NULL
+              AND b.lock_level != 'user'
+            ORDER BY
+                b.last_confirmed_at ASC NULLS FIRST,
+                b.last_retrieved_at ASC NULLS FIRST,
+                b.created_at ASC
+            LIMIT {int(limit)}
+            """
```

Ruff (0.15.15)

[error] Possible SQL injection vector through string-based query construction

(S608)